src/app/components/lane-dashboard/lane-dashboard.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="p-grid p-flex-column align-center">
+    <div class="p-col-10 p-lg-5 p-md-10">
+        <div class="p-grid">
+            <div class="p-col-12 p-lg-12 p-md-12 header">
+                <h3>Hey Aaron! Where would you like to begin?</h3>
+            </div>
+            <div class="p-col-12 p-lg-6 p-md-12">
```

Comment:
@SwathiChalumuri This particular div has repeated code. So, can we create it as separate component and use it here.

Reply:
ok I do that.